components/brave_wallet/browser/BUILD.gn

```diff
@@ -374,6 +378,22 @@ source_set("transaction") {
   ]
 }
 
+source_set("test_support") {
+  testonly = true
+  sources = [
+    "test_utils.cc",
+    "test_utils.h",
+  ]
+
+  deps = [
+    ":browser",
+    "//base",
+    "//components/value_store",
+    "//components/value_store:test_support",
+    "//testing/gmock",
+  ]
+}  # source_set("test_support")
```

Comment:
Strictliy speaking these
```
    "//base",
    "//components/value_store",
    "//components/value_store:test_support",
```
are referenced in pubic header so should be in `public_deps`

Reply:
fixed in 330c37ab4a9a518d77a58769424ffe8b45e84776